File: src/java/src/main/java/com/btgpactualsolutions/etd/model/EvaluateResult.java
package com.btgpactualsolutions.etd.model;

public class EvaluateResult {
	private final String issue;

	/**
	 * 
	 * @param issue
	 */
	public EvaluateResult(String issue) {
		this.issue = issue;
	}

	public boolean hasIssue() {
		return issue != null && issue.length() > 0;
	}

	public String getIssue() {
		return this.issue;
	}
}

File: src/dotnet/src/btg-solutions-etd/core/model/EvaluateResult.cs
namespace BtgPactualSolutions.Etd.core.model {
    public class EvaluateResult {
        private string _issue;
        internal EvaluateResult(string issue) {
            _issue = issue;
        }

        public bool HasIssue() {
            return !string.IsNullOrEmpty(_issue);
        }

        public string GetIssue() {
            return _issue;
        }
    }
}
